### PR TITLE
Feature/tree selection

### PR DESCRIPTION
**Pull request by jungining**
- 나무 생성 버튼 클릭 - > 나무 선택 Popup() -> 나무 선택 -> 선택된 prefab으로 나무 생성 로직 추가

### CODE REVIEW

File: Unity/PetEver/Assets/02.Scripts/TreeController.cs [hunk 1]
```diff
@@ -53,28 +61,84 @@ private void Awake()
...
+                    treeButtons[i].GetComponent<Image>().sprite = originalBtnSprite;
+                }
+            }
+        }
+        void LoadTreeResoures()
+        {
+            treePrefabs = new Dictionary<string, GameObject>();
+            treePrefabs.Add(treeName[0], Resources.Load<GameObject>("Prefabs/" + treeName[0]));
+            treePrefabs.Add(treeName[1], Resources.Load<GameObject>("Prefabs/" + treeName[1]));
+            treePrefabs.Add(treeName[2], Resources.Load<GameObject>("Prefabs/" + treeName[2]));
```
Comment (jungining) [inline Unity/PetEver/Assets/02.Scripts/TreeController.cs:139]: 이런곳은 좀 더러운데.. 더 나은 코드가 있으면 의견주세요

File: Unity/PetEver/Assets/02.Scripts/TreeController.cs [hunk 2]
```diff
@@ -53,28 +61,84 @@ private void Awake()
...
+                {
+                    createTree(selectedTree);
+                    selectedTree = null;
+                }
             });
+        }
 
+        void initTreeButtons()
+        {
+            int treeBtnSize = 3;
```
Comment (jeongchanKim) [inline Unity/PetEver/Assets/02.Scripts/TreeController.cs:101]: If this `treeBtnSize` is common for another function, it is better to declare it as global/static. Then it is much better to modify if want to change the size.

File: Unity/PetEver/Assets/02.Scripts/TreeController.cs [hunk 3]
```diff
@@ -83,24 +147,41 @@ void ShowPopUpDetail(CanvasGroup canvasGroup)
...
+            {
+                ExitPopupByCanvas(treeCreatePopupPanelCanvasGroup);
+            }
+            else if (treePopupCanvasGroup.interactable)
+            {
+                ExitPopupByCanvas(treePopupCanvasGroup);
+            }
+            else
+            {
+                Debug.Log("There isn't opend Popup");
```
Comment (jeongchanKim) [inline Unity/PetEver/Assets/02.Scripts/TreeController.cs:163]: `opend` -> `opended`.

**Review by jeongchanKim — APPROVED** [review]
Please check the comment, and apply them after.
Approved.

**Pull request merged**